nss-day-cohort-29/nutshell-spirit-wind
# Added NEws.js Login/session funcitionality

Description
news does not appear until logged in further more based on who is logged in the news will be user specific

  New feature (login functionality)
Testing Instructions for Change Made
git fetch --all
git checkout DAkota
cd src/lib
grunt
make sure that news does not appear until login and then based on who is loged in the news changes to who created it
Checklist:
  My code follows the style guidelines of this project Y
  I have performed a self-review of my own code N
  I have commented my code, particularly in hard-to-understand areas N
  My changes generate no new warnings N
  I have added test instructions that prove my fix is effective or that my feature works Y
